### PR TITLE
build: adding renku-minimal template

### LINKED ISSUE #59: Can an empty project be a minimal renku set-up
Would be it possible that the `Empty` project is the most skinny version of Renku project possible. What I mean is that if I'd like to use Renku neither for Python nor R (I used it for Bash), I would still expect that when I create an empty project through the Renku UI, I'll get a Renku project where I can use Renku commands like: `renku run ....`, `renku dataset create ...` and so on. What I'd imagine the project would have is the most basic Renku `Dockerfile`, `.gitlab-ci.yml`,  `.dockerignore`, `.gitignore` and probably `.renku` folder. Perhaps, the `Empty` project could be even named in a bit more descriptive way e.g. `Empty Renku project` but even if we keep the name as it is now, I believe it won't be that much misleading.

### PR DESCRIPTION
**Pull request by emmjab**
The `renku-minimal` template is for adding just enough dependencies to be able to launch a notebook on renkulab & use renku. It's a good option when you want to renku-ize an existing project and/or are using a language that's not R or python.

closes #59

### CODE REVIEW

**Comment by emmjab** [comment]
closes https://github.com/SwissDataScienceCenter/renku-project-template/issues/59

File: renku-minimal/.renku/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  created: schema:dateCreated
+  schema: http://schema.org/
+  name: schema:name
+  updated: schema:dateUpdated
+  version: schema:schemaVersion
+'@type': schema:Project
+created: {{ date_created }}
+name: {{ name }}
+updated: {{ date_updated }}
+version: '1'
```
Comment (rokroskar) [inline renku-minimal/.renku/metadata.yml]:
```suggestion
version: '3'
```

File: manifest.yaml [hunk 2]
```diff
@@ -6,3 +6,7 @@
   folder: R-minimal
   name: Basic R Project
   description: The simplest R-based renku project with a basic directory structure and necessary supporting files.
+-
+  folder: renku-minimal
```
Comment (rokroskar) [inline manifest.yaml]: should it just be called `minimal`? it has no language preference - the other two are `python-minimal` and `r-minimal`.

File: manifest.yaml [hunk 3]
```diff
@@ -6,3 +6,7 @@
...
+  name: Basic Renku and Renkulab-compatible Project
```
Comment (rokroskar) [inline manifest.yaml]: How about just "Minimal Renku project"

File: manifest.yaml (hunk 2 above)
Comment (emmjab) [reply manifest.yaml]: sure!

File: manifest.yaml (hunk 3 above)
Comment (emmjab) [reply manifest.yaml]: also ok! i'll fix.

**Pull request merged**